.storybook/main.js

```diff
@@ -76,6 +76,15 @@ module.exports = {
       'emotion-theming': toPath('node_modules/@emotion/react'),
     };
 
+    config.resolve.fallback = {
+      fs: false,
+      crypto: false,
+      browser: false,
+      tls: false,
+      https: false,
+    };
```

Comment:
Storybook wouldn't build unless I added this - it's basically a way of suppressing built-in node libraries in Webpack 🤷‍♀️